.github/workflows/upload-test-stats.yml

```diff
@@ -58,6 +58,14 @@ jobs:
           python3 -m tools.stats.upload_test_stats --workflow-run-id "${WORKFLOW_RUN_ID}" --workflow-run-attempt "${WORKFLOW_RUN_ATTEMPT}" --head-branch "${HEAD_BRANCH}"
           python3 -m tools.stats.upload_sccache_stats --workflow-run-id "${WORKFLOW_RUN_ID}" --workflow-run-attempt "${WORKFLOW_RUN_ATTEMPT}"
 
+      - name: Rerun if failed on master
```

Comment:
I would be hard, but retrying an actual failures would be pointless.  We could improve this a bit by reusing the logic in creating our alarm, i.e. https://github.com/pytorch/test-infra/issues/837 and skip all known failures.  Thoughts?